core/src/test/java/org/elasticsearch/search/aggregations/metrics/FilterTests.java

```diff
@@ -32,6 +32,7 @@ protected FilterAggregatorBuilder createTestAggregatorBuilder() {
         // NORELEASE make RandomQueryBuilder work outside of the
         // AbstractQueryTestCase
         // builder.query(RandomQueryBuilder.createQuery(getRandom()));
+        // TODO the norelease above
```

Comment:
These kind of scare me but I'm not sure the right way to address this. This PR is already too big.